src/main/kotlin/com/hedvig/accountservice/schedulers/domain/MemberScheduler.kt

```diff
@@ -389,6 +396,24 @@ class MemberScheduler {
         currentState = event.nextState
     }
 
+    @CommandHandler
+    fun handle(command: UnableToRequestChargeCommand) {
+        val nextState = SchedulerState(
+            memberId = command.memberId,
+            status = ChargeStatus.CHARGE_REQUEST_ERROR,
+            changedBy = command.scheduledBy,
+            amount = command.amount
+        )
+
+        apply(ChargeRequestErrorEvent(command.memberId, nextState))
+        logger.error { "Unable to request charge for memberId=${command.memberId}, exception: ${command.exception}" }
```

Comment:
Aparently you can do
```suggestion
        logger.error (command.exception) { "Unable to request charge for memberId=${command.memberId}" }
```